tox.ini

```diff
@@ -9,12 +9,17 @@ exclude = .git, .tox, ./lib, ./include, ./bin
 [testenv]
 deps =
   -r{toxinidir}/requirements.txt
+  -r{toxinidir}/requirements-lint.txt
+  -r{toxinidir}/requirements-test.txt
   -r{toxinidir}/requirements-dev.txt
 commands =
-    nosetests --with-coverage --cover-inclusive --cover-erase --cover-package=relayer --cover-html --rednose -s --cover-min-percentage=90
+    nosetests
 
 [testenv:flake8]
 deps =
+  -r{toxinidir}/requirements.txt
+  -r{toxinidir}/requirements-lint.txt
+  -r{toxinidir}/requirements-test.txt
```

Comment:
If you follow the suggestion I made on the `requirements-dev.txt` file, you'd only need to do `-r{toxinidir}/requirements-dev.txt`, because that one includes all the previous ones. (Not a blocker)